Packs/Grafana/Integrations/Grafana/Grafana.py

```diff
@@ -0,0 +1,799 @@
+        outputs_key_field='id',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown('Users', response, removeNull=True, headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def user_get_by_id_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    user_id = str(args.get('user_id'))
+
+    response = client.user_get_by_id_request(user_id)
+    command_results = CommandResults(
+        outputs_prefix='Grafana.User',
+        outputs_key_field='id',
+        outputs=response,
+        raw_response=response,
+        readable_output=tableToMarkdown(f'User {user_id} Results', response, removeNull=True, headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def users_teams_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    user_id = str(args.get('user_id'))
+
+    response = client.users_teams_request(user_id)
+    output = {'id': user_id, 'teams': response}
+
+    command_results = CommandResults(
+        outputs_prefix='Grafana.User',
+        outputs_key_field='id',
+        outputs=output,
+        raw_response=response,
+        readable_output=tableToMarkdown(f'Teams For User {user_id}', response, removeNull=True, headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def users_organization_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    user_id = str(args.get('user_id'))
+
+    response = client.users_organization_request(user_id)
+    output = {'id': user_id, 'orgs': response}
+
+    command_results = CommandResults(
+        outputs_prefix='Grafana.User',
+        outputs_key_field='id',
+        outputs=output,
+        raw_response=response,
+        readable_output=tableToMarkdown(f'Organization For User {user_id}', response, removeNull=True,
+                                        headerTransform=pascalToSpace)
+    )
+
+    return command_results
+
+
+def user_update_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    user_id = str(args.get('user_id'))
+    email = args.get('email')
+    name = args.get('name')
+    login = args.get('login')
+    theme = args.get('theme')
+
+    # login\email is required
+    if not login and not email:
+        raise DemistoException("Login or email must be filled.")
+
+    response = client.user_update_request(user_id, email, name, login, theme)
+    command_results = CommandResults(
+        readable_output=response['message']
+    )
+
+    return command_results
+
+
+def annotation_create_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    dashboard_id = args.get('dashboard_id')
+    panel_id = args.get('panel_id')
+
+    #  epoch numbers in millisecond resolution
+    time_start = args.get('time')
+    if time_start:
+        time_start = str(date_to_timestamp(time_start, DATE_FORMAT))
+    time_end = args.get('time_end')
+    if time_end:
+        time_end = str(date_to_timestamp(time_end, DATE_FORMAT))
```

Comment:
extract to a helper function please

Reply:
done, extracted to `set_time_for_annotation`